features/window-management.yml

```
@@ -0,0 +1,24 @@
+name: Window Management
```

[Sentence casing](https://github.com/web-platform-dx/web-features/blob/main/docs/guidelines.md#names), please

```suggestion
name: Window management
```